### ISSUE TITLE
PIXI/earcut hole punching throws error for certain grid sizes

### ISSUE THREAD

**Issue by ABrandau** [sPOiDar/fvtt-module-better-target#3]
Test case, target a token with a scale of 0,5 , the program screen will pseudo freeze while the UI components are still reachable, have to change scenes to reset or re focus the camera to solve the problem, small tokens are untargetable with the plugin enabled (or they are, but I dont get any visual feedback so I cant confirm)

**Comment by sPOiDar** [comment]
I cannot reproduce, do you have other modules installed, or alternative reproduction steps?
![Screenshot_20210416_101739](https://user-images.githubusercontent.com/70048780/114954032-053efc00-9e9d-11eb-92b3-875322a33c62.png)

**Comment by ABrandau** [comment]
Okay, I did some further testing as I tried to generate another situation to test this,

![image](https://user-images.githubusercontent.com/11763228/115112419-7e008f80-9f53-11eb-96f6-5aa99c4fad07.png)

This is the condition that is required on the token, or atleast the one that broke the program, now, as I targetted the token it functioned properly, so I went back to the original map to see if maybe an updated changed something but no, the bug remains there

After testing a few settings, I noticed the fault here is the grid size, a grid size of 64 (the original value) will not break the module but more difficult number like 72 will.

Setting the grid size to 72 and targetting the token will generate the bug, and its the same bug that i described on the earlier post

![test](https://user-images.githubusercontent.com/11763228/115112708-ca000400-9f54-11eb-8ef2-e274e00c7816.gif)

The scene i tested with had originally a grid size of 64, so I can suspect is about perfect roots, but otherwise my knowledge is limited.

Edit: Just tested other grid sizes, 72 seems to be the only unlucky number so far, so I can work around this.
